content/projects/infrastructure/index.adoc

```diff
@@ -81,7 +81,7 @@ Here is a non-exhaustive list of services that we provide and maintain.
 | Service                                                  | Issue tracker                                                                     | Repository and links
 | https://accounts.jenkins.io[Accounts]                    | https://github.com/jenkins-infra/helpdesk/labels/account[GitHub issues]           | -
 | https://accounts.jenkins.io[Account App]                 | https://github.com/jenkins-infra/helpdesk/labels/account-app[GitHub issues]       | https://github.com/jenkins-infra/account-app[Code],     https://github.com/jenkins-infra/charts/tree/master/charts/accountapp[helm chart]
-| link:/ircbot[Chatbot]                                    | https://github.com/jenkins-infra/helpdesk/labels/ircbot[GitHub issues]            | https://github.com/jenkins-infra/ircbot[Code],          https://github.com/jenkins-infra/charts/tree/master/charts/chatbot[helm chart]
+| link:./ircbot[Chatbot]                                   | https://github.com/jenkins-infra/helpdesk/labels/ircbot[GitHub issues]            | https://github.com/jenkins-infra/ircbot[Code],          https://github.com/jenkins-infra/charts/tree/master/charts/chatbot[helm chart]
```

Comment:
There isn't an ircbot link available at that location as far as I know.
```suggestion
| ircbot                                                   | https://github.com/jenkins-infra/helpdesk/labels/ircbot[GitHub issues]            | https://github.com/jenkins-infra/ircbot[Code],          https://github.com/jenkins-infra/charts/tree/master/charts/chatbot[helm chart]
```

Reply:
The link here works: https://deploy-preview-4966--jenkins-io-site-pr.netlify.app/projects/infrastructure/ , so I don't think the change is needed.